benchmark/benchmain/main.go

```diff
@@ -447,13 +441,21 @@ func makeFuncUnconstrainedStream(bf stats.Features) (rpcSendFunc, rpcRecvFunc, r
 		}, cleanup
 }
 
-func setupUnconstrainedStream(bf stats.Features) ([][]testgrpc.BenchmarkService_StreamingCallClient, *testpb.SimpleRequest, rpcCleanupFunc) {
+func setupStream(bf stats.Features, mode string) ([][]testgrpc.BenchmarkService_StreamingCallClient, *testpb.SimpleRequest, rpcCleanupFunc) {
 	clients, cleanup := makeClients(bf)
 
 	streams := make([][]testgrpc.BenchmarkService_StreamingCallClient, bf.Connections)
-	md := metadata.Pairs(benchmark.UnconstrainedStreamingHeader, "1",
-		benchmark.UnconstrainedStreamingDelayHeader, bf.SleepBetweenRPCs.String())
-	ctx := metadata.NewOutgoingContext(context.Background(), md)
+	ctx := context.Background()
+	if mode == workloadsUnconstrained {
+		md := metadata.Pairs(benchmark.UnconstrainedStreamingHeader, "1",
+			benchmark.UnconstrainedStreamingDelayHeader, bf.SleepBetweenRPCs.String())
```

Comment:
Nit: Please don't split up lines this way.

Please see line-length section in the Golang style guide: https://google.github.io/styleguide/go/guide#line-length.